visidata/basesheet.py

```diff
@@ -24,6 +25,15 @@ def keys(self):
     def clear(self):
         self.locals.clear()
 
+    def __len__(self):
+        return len(set().union(self.objs, self.locals))
+
+    def __contains__(self, key):
+        return key in self.objs or key in self.locals
+
+    def __iter__(self):
+        return iter(set().union(self.objs, self.locals))
```

Comment:
I think this is incorrect, because `self.objs` is `(k, obj)` instead of `(k, v)`.  Probably this should be an actual generator that loops over `union(self.objs.keys(), self.locals.keys())` and yields `k, self[k]`.